### PR TITLE
v0.10 changes

### PR DESCRIPTION
**Pull request by dhairyagandhi96**

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -1,3 +1,17 @@
...
+  - Tracker compatibility is maintained in most common cases, but Zygote will be the preferred AD backend for Flux from now on.
+* The CUDNN wrappers have been [moved from Flux into CuArrays](https://github.com/FluxML/Flux.jl/pull/874), to allow for better supporting the CUDA backend, and improve user experience, not to mention making Flux lean.
+* `*crossentropy` functions now [work as expected with CuArrays](https://github.com/FluxML/Flux.jl/pull/926). [PR for binarycrossentropy](https://github.com/FluxML/Flux.jl/pull/940).
+* Added a new [RADAM optimiser](https://github.com/FluxML/Flux.jl/pull/842)
+* Added [clearer docs](https://github.com/FluxML/Flux.jl/pull/904) around training and the Optimiser interface.
+* [Layer initialisations](https://github.com/FluxML/Flux.jl/pull/937) have been improved with a clearer API on how to extend it for other purposes.
+* [Better messaging around CUDA availability](https://github.com/FluxML/Flux.jl/pull/924), with hooks to initialize the GPU as default where possible.
+* @treelike has been formalised as a [functor](https://github.com/FluxML/Flux.jl/pull/865), with an effective deprecation.
```
Comment (MikeInnes) [inline NEWS.md]: Watch out for accidental `@mentions` here.

Comment (MikeInnes) [reply NEWS.md]:
```suggestion
* `@treelike` has been formalised as a [functor](https://github.com/FluxML/Flux.jl/pull/865), with an effective deprecation.
```

**Comment by MikeInnes** [comment]
Thanks @dhairyagandhi96!

bors r+

**Comment by MikeInnes** [comment]
I think you may as well do the version bump in `Project.toml` here too, then we can run bors again.

**Comment by dhairyagandhi96** [comment]
Done

**Comment by dhairyagandhi96** [comment]
bors try

**Comment by MikeInnes** [comment]
Merged into #946.

**Pull request closed without merging**